Makefile

```diff
@@ -71,14 +71,17 @@ lint-container: ## Lint the Symfony DI container
 lint-twig: ## Lint Twig files
 	@bin/console lint:twig templates/
 
+fix-twig: ## Fix Twig files with Twig CS Fixer
+	@vendor/bin/twig-cs-fixer lint --fix ./templates
+
 lint-yaml: ## Lint YAML files
 	@bin/console lint:yaml --parse-tags config/
 
 cs: ## Run all CS checks
 cs: fix-php stan
```

Comment:
fix-twig might be added here, it say `## Run all CS checks` but say nothing bout php file only (but file weird having stan in the list of CS checks)

Reply:
Right! There are inconsistencies. It's clearer to me now. 
In fact, there are two types of checks:

* The “linters”, they only report problems
* The “fixers”, they also report problems, but they can also fix some of them

PHPStan is, in fact, a linter, so it's weird to have it in the “cs” namespace.

So the new groups would be:

* Linters : all + php-cs-fixer without the `--fix` flag, twig-cs-fixer without the `--fix flag`
* Fixers : php-cs-fixer, twig-cs-fixer (rector would also be a fixer)

WDYT? I will create a dedicated PR against for this cleanup. 
Thanks for the advices. 👍

Reply:
https://github.com/strangebuzz/MicroSymfony/pull/120

Reply:
If you want to be consistent you

Keep lint-twig for sf command
You rename lint-php to cs-php
You keep fix-php
You create cs-twig and fix-twig for twig csfixer